src/db.js

```diff
@@ -26,9 +26,22 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '314159',
```

Comment:
Ensure that the default password '314159' is secure and appropriate for your environment, especially in production settings.